src/posts/experimental-static-build.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Scaling Astro Sites to Thousands of Pages'
+description: 'A new experimental flag in Astro build enables building sites with tens of thousands of pages.'
+publishDate: 'January 25, 2022'
+authors:
+  - matthew
+lang: 'en'
+---
+
+Astro's about to get a lot faster with a new build system intended to scale to tens, or even hundreds, of thousands of pages. If you hang out in our [Discord](https://astro.build/chat) or pay attention to recent releases you might have seen a lot of discussion about a "static build". This is our new implementation of `astro build` that does 2 things:
```

Comment:
For a short, informational post like this it's good to get to the point at the start, in 1-2 sentences. Next.js does a good job of this in their blog posts, for example: https://nextjs.org/blog/next-12

Something like:

```
Astro is about to get a lot faster! Our new build optimization process is ready to try out in Astro today:

'''
astro build --experimental-static-build
'''

What's new? ... (this is where you can get into the prose that you have in the first paragraph).
```